src/ntfs/mod.rs

```diff
@@ -170,6 +172,18 @@ pub fn write<P: AsRef<path::Path>>(model: &Model, path: P) -> Result<()> {
     Ok(())
 }
 
+/// Exports a `Model` to a
+/// [NTFS](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fr.md)
+/// ZIP archive at the given full path.
+pub fn write_to_zip<P: AsRef<path::Path>>(model: &Model, path: P) -> Result<()> {
+    let path = path.as_ref();
+    info!("Writing NTFS to ZIP File {:?}", path);
+    let input_tmp_dir = TempDir::new("read_netex_work").unwrap();
+    write(model, input_tmp_dir.path())?;
+    zip_to(input_tmp_dir.path(), path)?;
+    Ok(())
+}
```

Comment:
I thought about this, but trying to achieve this would implies a heavy refacto and/or a lot of time to do it. The main purpose of this PR is to enable a Netex reading. It could be an enhancement on a further PR.